app/src/main/res/values-w600dp/dimens.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <dimen name="fab_margin">48dp</dimen>
+</resources>
```

Comment:
What's up with the changes to the Android resources? Consider backing out these changes if they're not relevant - otherwise it might create a conflict if someone else is working on the same file?